internal/infrastructure/mongo/mongodoc/user.go

```diff
@@ -56,6 +65,11 @@ func NewUser(user *user1.User) (*UserDocument, string) {
 		Team:         user.Team().String(),
 		Lang:         user.Lang().String(),
 		Theme:        string(user.Theme()),
+		Verification: &UserVerificationDoc{
+			Code:       user.Verification().Code(),
+			Expiration: user.Verification().Expiration(),
+			Verified:   user.Verification().IsVerified(),
+		},
```

Comment:
Verification may not always be present; if it is nil, do not simply create a `UserVerificationDoc`.